master/internal/api_checkpoint.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/determined-ai/determined/master/internal/api"
 	"github.com/determined-ai/determined/master/internal/authz"
 	"github.com/determined-ai/determined/master/internal/checkpoints"
-	"github.com/determined-ai/determined/master/internal/db"
+	internaldb "github.com/determined-ai/determined/master/internal/db"
```

Comment:
named import as `internaldb` to differentiate between db the type and db the package import; I think this is more clear (especially since `db` is sometimes used in multiple contexts in the same file) but can undo it if it's against style guidelines